lib/oli/delivery/attempts/manual_grading.ex

```diff
@@ -260,32 +283,52 @@ defmodule Oli.Delivery.Attempts.ManualGrading do
 
     now = DateTime.utc_now()
 
-    Enum.filter(part_attempts, fn pa -> pa.grading_approach == :manual and pa.lifecycle_state == :submitted end)
+    Enum.filter(part_attempts, fn pa ->
+      pa.grading_approach == :manual and pa.lifecycle_state == :submitted
+    end)
     |> Enum.reduce_while({:ok, []}, fn pa, {:ok, updated} ->
       case Map.get(score_feedbacks_map, pa.attempt_guid) do
         %{score: score, out_of: out_of, feedback: feedback} ->
-          case Core.update_part_attempt(pa, %{lifecycle_state: :evaluated, date_evaluated: now, score: score, out_of: out_of, feedback: %{content: wrap_in_paragraphs(feedback)}}) do
+          case Core.update_part_attempt(pa, %{
+                 lifecycle_state: :evaluated,
+                 date_evaluated: now,
+                 score: score,
+                 out_of: out_of,
+                 feedback: %{content: wrap_in_paragraphs(feedback)}
+               }) do
             {:ok, updated_part_attempt} -> {:cont, {:ok, [updated_part_attempt | updated]}}
             e -> {:halt, e}
           end
-        _ -> {:halt, {:error, "scoring feedback not found for attempt #{pa.attempt_guid}"}}
+
+        _ ->
+          {:halt, {:error, "scoring feedback not found for attempt #{pa.attempt_guid}"}}
       end
     end)
-
   end
 
-  defp maybe_finalize_resource_attempt(_, false, resource_attempt_guid), do: {:ok, resource_attempt_guid}
+  defp maybe_finalize_resource_attempt(_, false, resource_attempt_guid),
+    do: {:ok, resource_attempt_guid}
 
   defp maybe_finalize_resource_attempt(section, true, resource_attempt_guid) do
-    case Oli.Delivery.Attempts.PageLifecycle.Graded.roll_up_activities_to_resource_attempt(resource_attempt_guid) do
+    case Oli.Delivery.Attempts.PageLifecycle.Graded.roll_up_activities_to_resource_attempt(
+           resource_attempt_guid
+         ) do
       {:ok, %ResourceAttempt{lifecycle_state: :evaluated, resource_access_id: resource_access_id}} ->
-        Oli.Delivery.Attempts.PageLifecycle.Graded.roll_up_resource_attempts_to_access(section.slug, resource_access_id)
+        Oli.Delivery.Attempts.PageLifecycle.Graded.roll_up_resource_attempts_to_access(
+          section.slug,
+          resource_access_id
+        )
         |> maybe_initiate_grade_passback(section)
-      e -> e
+
+      e ->
+        e
     end
   end
 
-  defp maybe_initiate_grade_passback({:ok, %ResourceAttempt{resource_access_id: resource_access_id}}, %Section{id: section_id, grade_passback_enabled: true}) do
+  defp maybe_initiate_grade_passback({:ok, %ResourceAccess{id: resource_access_id}}, %Section{
```

Comment:
This is the only real change.  Corrects the pattern match to target ResourceAccess.